vllm/platforms/interface.py

```diff
@@ -543,6 +543,19 @@ def stateless_init_device_torch_dist_pg(
         """
         raise RuntimeError(f"Unsupported torch distributed backend: {backend}")
 
+    @classmethod
+    def is_kv_cache_dtype_supported(cls, kv_cache_dtype: str) -> bool:
+        """
+        Returns if the kv_cache_dtype is supported by the current platform.
+        """
+        import vllm.envs as envs
+        fp8_attention = kv_cache_dtype.startswith("fp8")
+        will_use_fa = envs.VLLM_ATTENTION_BACKEND == "FLASH_ATTN_VLLM_V1"
+        if fp8_attention and will_use_fa:
+            from vllm.attention.utils.fa_utils import flash_attn_supports_fp8
+            return flash_attn_supports_fp8()
+        return False
```

Comment:
I think the base class should always return false so it is up to the specific impls to define
```suggestion
        return False
```

Reply:
@mgoin. Have updated codes based your suggestion